backend/src/main/java/com/skr/virtuallibrary/mapping/ModelMapper.java

```diff
@@ -28,12 +28,15 @@ public interface ModelMapper {
 
     BookRating toBookRatingEntity(BookRatingDto bookRatingDto);
 
-    @Mapping(target = "created", source = "auditData.createdDate")
-    @Mapping(target = "lastModified", source = "auditData.lastModifiedDate")
-    ReviewDto toReviewDto(Review review);
+    @Mapping(target = "created", source = "review.auditData.createdDate")
+    @Mapping(target = "lastModified", source = "review.auditData.lastModifiedDate")
```

Comment:
Jak już to zmieniamy, to może w DTO też powinien być obiekt AuditData, zamiast mapowania poszczególnych pól?

Reply:
tak jak ci mówiłem wątpię, że te pola będą przydatne, ale jak chcesz (lub frontend), to mogę to zmienić